tokio-postgres/src/config.rs

```diff
@@ -153,6 +169,7 @@ pub struct Config {
     pub(crate) application_name: Option<String>,
     pub(crate) ssl_mode: SslMode,
     pub(crate) host: Vec<Host>,
+    pub(crate) hostaddr: Vec<String>,
```

Comment:
I think it'd be good to store these as `IpAddr`s and have the setter take that type as well.

Reply:
Sure. Will do.